accounts/abi/bind/bind.go

```diff
@@ -140,9 +140,10 @@ func Bind(types []string, abis []string, bytecodes []string, fsigs []map[string]
 			normalized.Outputs = make([]abi.Argument, len(original.Outputs))
 			copy(normalized.Outputs, original.Outputs)
 			for j, output := range normalized.Outputs {
-				if output.Name != "" {
-					normalized.Outputs[j].Name = capitalise(output.Name)
+				if output.Name == "" {
+					return "", fmt.Errorf("missing output name")
```

Comment:
The issue here is that if there is no name, when we generate the precompile code the output is un-named and leads to an error.

I'd prefer to simply require that all of the outputs are named to make it more clear. Additionally, we should make this error more clear, so that a user that sees it knows exactly what they need to change.

We could say `fmt.Errorf("ABI outputs for %s require a name to generate the precompile binding, re-generate the ABI from a Solidity source file with all named outputs", normalized.Name)`

Reply:
We should not change the default abigen behaviour though, let's return an error only if this is precompile gen.

Reply:
Good point, sounds good to me

Reply:
thanks